### PR TITLE
CC-3797 add updates field in audit log

### PR DESCRIPTION
**Pull request by ssOlexBaiko**
[Jira](https://jira.zenoss.com/browse/CC-3797)

### CODE REVIEW

**Comment by webhookbotuser** [comment]
Build finished.

**Comment by ssOlexBaiko** [comment]
retest this

File: facade/service.go [hunk 1]
```diff
@@ -257,6 +258,39 @@ func validateServiceOptions(svc *service.Service) error {
 	return nil
 }
 
+//Get changes made by "serviced service edit" call
+func (f * Facade) getChanges(ctx datastore.Context, svc service.Service) string {
```
Comment (tgjonestx) [inline facade/service.go]: Please add a unit or integration test for this.

File: facade/service_test.go [hunk 2]
```diff
@@ -3770,3 +3770,9 @@ func (ft *FacadeIntegrationTest) assertPathResolvesToServices(c *C, path string,
 	c.Assert(foundids, DeepEquals, ids)
 }
 
+func (ft *FacadeIntegrationTest) TestFacade_getChanges(c *C) {
```
Comment (tgjonestx) [inline facade/service_test.go]: This test verifies the edge-case where there are no changes.
Can you add a test case that verifies the main part of `getChanges()`?  In other words, something that shows it found an actual change?
